src/main/scala/is/hail/io/vcf/LoadVCF.scala

```diff
@@ -812,36 +816,69 @@ object LoadVCF {
     }
   }
 
-  def apply(hc: HailContext,
-    reader: HtsjdkRecordReader,
+  def pyApply(
+    files: java.util.ArrayList[String],
+    callFields: java.util.ArrayList[String],
+    headerFile: String,
+    minPartitions: Option[Int],
+    dropSamples: Boolean,
+    rg: ReferenceGenome,
+    contigRecoding: java.util.Map[String, String],
+    arrayElementsRequired: Boolean,
+    skipInvalidLoci: Boolean,
+    gzAsBGZ: Boolean,
```

Comment:
`gzAsBGZ` is the function, confusing to re-use as Boolean. Should it still be called `forceBGZ`?

Reply:
this parameter name is WAY clearer. I've renamed the function.